report-ng/app/src/components/log-view/abstract-log-view.ts

```diff
@@ -37,11 +40,27 @@ export abstract class AbstractLogView {
     @bindable({defaultBindingMode: bindingMode.toView})
     searchRegexp:RegExp;
 
+    //improve performance
+    //The cache uses the timestamp as the key and the formatted timestamp as the value.
+    private timestampCache: Map<number, string> = new Map<number, string>();
```

Comment:
Yeah... the cache seems to be help to improve the performance to a acceptable level.